boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuapp_0_7_0.overlay

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54l15pdk_nrf54l15_common_0_3_0.dtsi"
```

Comment:
@barsok if 0.7.0 has the same pinout for LEDs and buttons as 0.3.0, then file `nrf54l15pdk_nrf54l15_common_0_3_0.dtsi` should most probably be renamed to something that will indicate that it is common for 0.3.0 and 0.7.0, for example, `nrf54l15pdk_nrf54l15_common_0_3_0_S_0_7_0.dtsi` where 'S' stands for '/' (it does not look good, but it is the best one I came up with for now). Otherwise, the fact that 0.7.0 overlay includes dtsi for 0.3.0 may be confusing for some people.

Reply:
Thank you for this suggestion. I have added comment for 0.7.0, maybe this is acceptable? Checked other vendors and there are cases only for "ns" environment where the same file is included. Here the 0.7.0 acts as a label for 0.3.0.